packages/runtime/container-runtime/garbageCollection.md

```diff
@@ -1,28 +1,38 @@
 # Garbage Collection
-Garbage collection (GC) identifies Fluid objects that are not used and deletes them from the Fluid document. This reduces the size of the Fluid file, the in-memory content and the summary that is uploaded to / downloaded from the server. It also makes processing faster as there is less data to process.
+Garbage collection (GC) is the process by which Fluid Framework performs automatic memory management. GC will run periodically and safely delete objects that are not used. The only responsibility of the users of Fluid Framework is to add and remove references to Fluid objects correctly.
```

Comment:
I wouldn't say "memory management" because that calls to mind RAM/working set.  FF already doesn't load stuff that's not used in the doc (although it does have to download it serialized in the snapshot...).